app/gui/src/presenter/graph/state.rs

```diff
@@ -194,7 +194,7 @@ impl Expressions {
     /// Remove all expressions not belonging to the any of the `nodes`.
     pub fn retain_expression_of_nodes(&mut self, nodes: &HashSet<AstNodeId>) {
         let nodes_to_remove =
-            self.expressions_of_node.drain_filter(|node_id, _| !nodes.contains(node_id));
+            self.expressions_of_node.extract_if(|node_id, _| !nodes.contains(node_id));
```

Comment:
this is such a strange name. Im surprised. `drain_filter` was much nicer. Is it removed? :O

Reply:
Is it really? I think `drain` was clear to us, because we know what `drain` means in this context. But it's not obvious.

I actually think `extract` is a better word for `remove some elements from the vector and return them`.

Reply:
It is removed in the latest versions of the compiler. I will have to revert these changes though, because we need to return back to the previous compiler version.